controllers/storagecluster/platform_detection.go

```diff
@@ -27,7 +27,7 @@ var SkipObjectStorePlatforms = []configv1.PlatformType{
 	configv1.AWSPlatformType,
 	configv1.GCPPlatformType,
 	configv1.AzurePlatformType,
-	IBMCloudCosPlatformType,
+	configv1.IBMCloudCosPlatformType,
```

Comment:
I have changed and pushed changes

Reply:
I mean both of them should be added to the list so that both of them will be ignored.
```
configv1.IBMCloudCosPlatformType
configv1.IBMCloudPlatformType
```

Reply:
I have added both of them to the list